src/index.ts

```diff
@@ -1,9 +1,19 @@
-import { FastifyRequest, FastifyReply } from 'fastify'
-import { ServerResponse } from 'http'
+import { FastifyInstance, FastifyRequest, FastifyReply } from 'fastify'
 import fp from 'fastify-plugin'
+import { isInterference } from 'interference'
+
+function errorHandler(
+  this: FastifyInstance,
+  error: any,
```

Comment:
I think that the best thing is to add `Error` as type since it will be surely error or something that extends it.

Reply:
We cannot be sure that `error` is an instance of `Error`. For example, a `throw null` is possible and It will break this code.